courses/free-courses-hi.md

```diff
@@ -191,6 +191,7 @@
 
 * [Cloud Computing](https://www.youtube.com/playlist?list=PLYwpaL_SFmcCyQH0n9GHfwviu6KeJ46BV) - 5 Minutes Engineering
 * [Cloud Computing Tutorial](https://www.youtube.com/playlist?list=PL-JvKqQx2AtfQ8cGyKsFE7Tj2FyB1yCkd) - University Academy
+* [Cloud Computing Tutorial For Beginners](https://www.youtube.com/playlist?list=PL9ooVrP1hQOFtZ5oAAeOgi_nH-txMcDMu) - Edureka
```

Comment:
Move to "courses/free-courses-en.md" and use the same title given by the resource